src/modules/jackrack/CMakeLists.txt

```diff
@@ -21,6 +21,11 @@ if(USE_LV2)
   install(FILES filter_lv2.yml producer_lv2.yml DESTINATION ${MLT_INSTALL_DATA_DIR}/jackrack)
 endif()
 
+if(USE_VST2)
```

Comment:
This also depends on `GPL`

Reply:
So I remove GPL check for everything including the old ladspa code?

Reply:
No, I mean this line should be `if(USE_VST2 AND GPL)`. It should also have been `if(USE_LV2 AND GPL)`. This module has a mixed license. Only the `jack` consumer is LGPL while everything else is GPLv2. So, just as below we do not build those components unless `GPL` is on, it should not install the YAML files for these components unless `GPL` is on.